index.bs

```diff
@@ -87,20 +87,20 @@ interface VRDisplay : EventTarget {
   readonly attribute DOMString displayName;
 
   /**
+   * Populates the passed VRFrameData with the information required to render
+   * the current frame.
+   */
+  void getCurrentFrameData(VRFrameData frameData);
```

Comment:
Yes, it does! :D

On Wed, Sep 7, 2016 at 10:27 PM Justin Rogers <notifications@github.com>
wrote:

> In index.bs <https://github.com/w3c/webvr/pull/80#discussion_r77947756>:
>
> > @@ -87,20 +87,20 @@ interface VRDisplay : EventTarget {
> >    readonly attribute DOMString displayName;
> >
> >    /**
> > +   * Populates the passed VRFrameData with the information required to render
> > +   * the current frame.
> > +   */
> > +  void getCurrentFrameData(VRFrameData frameData);
>
> Agree on both fronts. And I'm testing if responding to these mails will
> auto comment for me. :-)
> … <#m_-3546350149252576669_>
> On Sep 7, 2016, at 10:04 PM, Brandon Jones ***@***.***> wrote: In index.bs:
> > @@ -87,20 +87,20 @@ interface VRDisplay : EventTarget { > readonly
> attribute DOMString displayName; > > /** > + * Populates the passed
> VRFrameData with the information required to render > + * the current
> frame. > + */ > + void getCurrentFrameData(VRFrameData frameData); The more
> code and docs I write about this the more this function name feels overly
> verbose to me. Anyone object to simply getFrameData()? Also, I think it
> should return a bool to indicate whether or not the frame data was
> populated. — You are receiving this because you commented. Reply to this
> email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/w3c/webvr/pull/80/files/e6c83cb18cccf995d70b83ae2e0add6ef3b7a59d#r77947756>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAxJma1Zdg-Ny7aYJy9iIvbzrpirp3_nks5qn5zRgaJpZM4J2KLS>
> .
>